search/search_indexes.py

```diff
@@ -137,6 +136,15 @@ def prepare_preview_url(self, obj):  # pylint: disable=no-self-use
             run=data["run"],
         )
 
+    def prepare_titlesort(self, obj):  # pylint: disable=no-self-use
+        """Define what goes into the "titlesort" index."""
+        title = obj.title.strip()
+        # hack to handle empty titles
+        # to show up at the bottom of the sorted list
+        if not title:
+            title = 'z' * 30
```

Comment:
To clarify, I mean relying on 'z' having a high value relative to everything else. It's reasonable to expect '0' to be before '1' and 'a' to be before 'z'